plugins/permission-policy-simple/src/SimplePermissionPolicy.ts

```diff
@@ -59,15 +59,15 @@ export class SimplePermissionPolicy implements PermissionPolicy {
       if (request.permission.attributes.action === 'read') {
         return createCatalogPolicyDecision({
           anyOf: [
-            isEntityOwner(getIdentityClaims(identity)),
+            isEntityOwner(getIdentityClaims(identity.identity)),
```

Comment:
`identity.identity` is a bit awkward. should the param for `handle` be changed to `identityResponse` or something? or to instead receive a `BackstageUserIdentity` directly and not a `BackstageIdentityResponse`?